internal/creds.go

```diff
@@ -226,3 +226,12 @@ func baseTransport() *http.Transport {
 		ExpectContinueTimeout: 1 * time.Second,
 	}
 }
+
+func ErrUniverseNotMatch(settingsUD, credsUD string) error {
+	return fmt.Errorf(
+		"the configured universe domain (%q) does not match the universe "+
+			"domain found in the credentials (%q). If you haven't configured "+
+			"WithUniverseDomain explicitly, googleapis.com is the default",
```

Comment:
Nit, all other universe domain values are wrapped in double quotes - should `googleapis.com` be as well?

Reply:
Fixed, thanks!